source/module_md/Makefile.Objects

```diff
@@ -31,7 +31,6 @@ global_function.o\
 global_variable.o\
 tool_title.o\
 tool_quit.o\
-mymath3.o\
```

Comment:
What's the purpose here?

Reply:
This source file is not needed by module_md.